feat: add `ndarray/base/nullary-strided1d`

headlessNode opened the pull request:
---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes. report:
  - task: lint_filenames status: passed
  - task: lint_editorconfig status: passed
  - task: lint_markdown status: passed
  - task: lint_package_json status: passed
  - task: lint_repl_help status: passed
  - task: lint_javascript_src status: passed
  - task: lint_javascript_cli status: na
  - task: lint_javascript_examples status: passed
  - task: lint_javascript_tests status: passed
  - task: lint_javascript_benchmarks status: na
  - task: lint_python status: na
  - task: lint_r status: na
  - task: lint_c_src status: na
  - task: lint_c_examples status: na
  - task: lint_c_benchmarks status: na
  - task: lint_c_tests_fixtures status: na
  - task: lint_shell status: na
  - task: lint_typescript_declarations status: na
  - task: lint_typescript_tests status: na
  - task: lint_license_headers status: passed ---

Progresses #2656.

## Description

> What is the purpose of this pull request?

This pull request:

-   add `ndarray/base/nullary-strided1d`

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   progresses #2656

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

headlessNode committed:
feat: add ndarray/base/nullary-strided1d

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: passed
  - task: lint_repl_help
    status: passed
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: passed
  - task: lint_javascript_tests
    status: passed
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/1d.js:
```diff
@@ -0,0 +1,169 @@
+* nullary1d( wrapper, [ x, sortOrder ], views, [ 3 ], [ 4 ], strategy, {} );
+*
+* var arr = ndarray2array( x.data, x.shape, x.strides, x.offset, x.order );
+* // returns [ [ [ 9, 10 ], [ 11, 12 ] ], [ [ 5, 6 ], [ 7, 8 ] ], [ [ 1, 2 ], [ 3, 4 ] ] ]
```
```suggestion
* // returns [ [ [ 9.0, 10.0 ], [ 11.0, 12.0 ] ], [ [ 5.0, 6.0 ], [ 7.0, 8.0 ] ], [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ]
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d.js:
```diff
@@ -0,0 +1,201 @@
+* nullary2d( wrapper, [ x, sortOrder ], views, [ 1, 3 ], [ 12, 4 ], true, strategy, {} );
+*
+* var arr = ndarray2array( x.data, x.shape, x.strides, x.offset, x.order );
+* // returns [ [ [ [ 9, 10 ], [ 11, 12 ] ], [ [ 5, 6 ], [ 7, 8 ] ], [ [ 1, 2 ], [ 3, 4 ] ] ] ]
```
```suggestion
* // returns [ [ [ [ 9.0, 10.0 ], [ 11.0, 12.0 ] ], [ [ 5.0, 6.0 ], [ 7.0, 8.0 ] ], [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ]
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d_blocked.js:
```diff
@@ -0,0 +1,231 @@
+* blockednullary2d( wrapper, [ x, sortOrder ], views, [ 1, 3 ], [ 12, 4 ], strategy, {} );
+*
+* var arr = ndarray2array( x.data, x.shape, x.strides, x.offset, x.order );
+* // returns [ [ [ [ 9, 10 ], [ 11, 12 ] ], [ [ 5, 6 ], [ 7, 8 ] ], [ [ 1, 2 ], [ 3, 4 ] ] ] ]
```
```suggestion
* // returns [ [ [ [ 9.0, 10.0 ], [ 11.0, 12.0 ] ], [ [ 5.0, 6.0 ], [ 7.0, 8.0 ] ], [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ]
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d.js:
```diff
@@ -0,0 +1,217 @@
+* nullary3d( wrapper, [ x, sortOrder ], views, [ 1, 1, 3 ], [ 12, 12, 4 ], true, strategy, {} );
+*
+* var arr = ndarray2array( x.data, x.shape, x.strides, x.offset, x.order );
+* // returns [ [ [ [ [ 9, 10 ], [ 11, 12 ] ], [ [ 5, 6 ], [ 7, 8 ] ], [ [ 1, 2 ], [ 3, 4 ] ] ] ] ]
```
```suggestion
* // returns [ [ [ [ [ 9.0, 10.0 ], [ 11.0, 12.0 ] ], [ [ 5.0, 6.0 ], [ 7.0, 8.0 ] ], [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ]
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d_blocked.js:
```diff
@@ -0,0 +1,256 @@
+* blockednullary3d( wrapper, [ x, sortOrder ], views, [ 1, 1, 3 ], [ 12, 12, 4 ], strategy, {} );
+*
+* var arr = ndarray2array( x.data, x.shape, x.strides, x.offset, x.order );
+* // returns [ [ [ [ [ 9, 10 ], [ 11, 12 ] ], [ [ 5, 6 ], [ 7, 8 ] ], [ [ 1, 2 ], [ 3, 4 ] ] ] ] ]
```
```suggestion
* // returns [ [ [ [ [ 9.0, 10.0 ], [ 11.0, 12.0 ] ], [ [ 5.0, 6.0 ], [ 7.0, 8.0 ] ], [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ]
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/factory.js:
```diff
@@ -0,0 +1,172 @@
+* sorthp( [ x, sortOrder ], [ 2, 3 ] );
+*
+* var arr = ndarray2array( x.data, x.shape, x.strides, x.offset, x.order );
+* // returns [ [ [ [ 9, 10 ], [ 11, 12 ] ], [ [ 5, 6 ], [ 7, 8 ] ], [ [ 1, 2 ], [ 3, 4 ] ] ] ]
```
```suggestion
* // returns [ [ [ [ 9.0, 10.0 ], [ 11.0, 12.0 ] ], [ [ 5.0, 6.0 ], [ 7.0, 8.0 ] ], [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ]
```

headlessNode committed:
docs: apply suggestions from code review

Signed-off-by: Muhammad Haris <101793258+headlessNode@users.noreply.github.com>

headlessNode committed:
refactor: refactor reshape strategy for nullary

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: passed
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

headlessNode committed:
Merge branch 'nullary-strided1d' of https://github.com/headlessNode/stdlib into nullary-strided1d

headlessNode committed:
feat: add remaining kernels

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/README.md:
```diff
@@ -0,0 +1,214 @@
+
+# nullaryStrided1d
+
+> Apply a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
> Apply a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/README.md:
```diff
@@ -0,0 +1,214 @@
+
+#### nullaryStrided1d( fcn, arrays, dims\[, options] )
+
+Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/README.md:
```diff
@@ -0,0 +1,214 @@
+// Define the index offsets:
+var ox = 0;
+
+// Create an input ndarray-like object:
```
```suggestion
// Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/README.md:
```diff
@@ -0,0 +1,214 @@
+
+The function accepts the following arguments:
+
+-   **fcn**: function which will be applied to a one-dimensional input subarray.
```
```suggestion
-   **fcn**: function which will be applied to a one-dimensional output subarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/README.md:
```diff
@@ -0,0 +1,214 @@
+
+    where
+
+    -   **arrays**: array containing a one-dimensional subarray of the input ndarray and any additional ndarray arguments as zero-dimensional ndarrays.
```
```suggestion
    -   **arrays**: array containing a one-dimensional subarray of the output ndarray and any additional ndarray arguments as zero-dimensional ndarrays.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/README.md:
```diff
@@ -0,0 +1,214 @@
+    -   **arrays**: array containing a one-dimensional subarray of the input ndarray and any additional ndarray arguments as zero-dimensional ndarrays.
+    -   **options**: function options (_optional_).
+
+-   The function iterates over ndarray elements according to the memory layout of the input ndarray.
```
```suggestion
-   The function iterates over ndarray elements according to the memory layout of the output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/docs/repl.txt:
```diff
@@ -0,0 +1,92 @@
+
+{{alias}}( fcn, arrays, dims[, options] )
+    Applies a one-dimensional strided array function to a list of specified
+    dimensions in an input ndarray.
```
```suggestion
    Applies a one-dimensional strided array function to a list of specified
    dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/docs/repl.txt:
```diff
@@ -0,0 +1,92 @@
+    Parameters
+    ----------
+    fcn: Function
+        Function which will be applied to a one-dimensional input subarray. The
```
```suggestion
        Function which will be applied to a one-dimensional output subarray. The
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/docs/repl.txt:
```diff
@@ -0,0 +1,92 @@
+
+        where
+
+        - arrays: array containing a one-dimensional subarray of the input
```
```suggestion
        - arrays: array containing a one-dimensional subarray of the output
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/docs/repl.txt:
```diff
@@ -0,0 +1,92 @@
+        - options: function options.
+
+    arrays: ArrayLikeObject<ndarray>
+        Array-like object containing an input ndarray followed by any additional
```
```suggestion
        Array-like object containing an output ndarray followed by any additional
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/0d.js:
```diff
@@ -0,0 +1,106 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/0d.js:
```diff
@@ -0,0 +1,106 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/0d.js:
```diff
@@ -0,0 +1,106 @@
+*     'order': 'row-major'
+* };
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d.js:
```diff
@@ -0,0 +1,327 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/0d.js:
```diff
@@ -0,0 +1,106 @@
+* @private
+* @param {Function} fcn - wrapper for a one-dimensional strided array function
+* @param {Array<Object>} arrays - ndarrays
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d.js:
```diff
@@ -0,0 +1,327 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d.js:
```diff
@@ -0,0 +1,327 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d.js:
```diff
@@ -0,0 +1,327 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d.js:
```diff
@@ -0,0 +1,327 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d_blocked.js:
```diff
@@ -0,0 +1,413 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d_blocked.js:
```diff
@@ -0,0 +1,413 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d_blocked.js:
```diff
@@ -0,0 +1,413 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d_blocked.js:
```diff
@@ -0,0 +1,413 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/10d_blocked.js:
```diff
@@ -0,0 +1,413 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/1d.js:
```diff
@@ -0,0 +1,165 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/1d.js:
```diff
@@ -0,0 +1,165 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/1d.js:
```diff
@@ -0,0 +1,165 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/1d.js:
```diff
@@ -0,0 +1,165 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/1d.js:
```diff
@@ -0,0 +1,165 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d.js:
```diff
@@ -0,0 +1,197 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d.js:
```diff
@@ -0,0 +1,197 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d.js:
```diff
@@ -0,0 +1,197 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d.js:
```diff
@@ -0,0 +1,197 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d.js:
```diff
@@ -0,0 +1,197 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d_blocked.js:
```diff
@@ -0,0 +1,227 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop-blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop-blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d_blocked.js:
```diff
@@ -0,0 +1,227 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d_blocked.js:
```diff
@@ -0,0 +1,227 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d_blocked.js:
```diff
@@ -0,0 +1,227 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d_blocked.js:
```diff
@@ -0,0 +1,227 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d.js:
```diff
@@ -0,0 +1,213 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d.js:
```diff
@@ -0,0 +1,213 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d.js:
```diff
@@ -0,0 +1,213 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d.js:
```diff
@@ -0,0 +1,213 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d.js:
```diff
@@ -0,0 +1,213 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d_blocked.js:
```diff
@@ -0,0 +1,252 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d_blocked.js:
```diff
@@ -0,0 +1,252 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d_blocked.js:
```diff
@@ -0,0 +1,252 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d_blocked.js:
```diff
@@ -0,0 +1,252 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/3d_blocked.js:
```diff
@@ -0,0 +1,252 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d.js:
```diff
@@ -0,0 +1,229 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d.js:
```diff
@@ -0,0 +1,229 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d.js:
```diff
@@ -0,0 +1,229 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d.js:
```diff
@@ -0,0 +1,229 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d.js:
```diff
@@ -0,0 +1,229 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d_blocked.js:
```diff
@@ -0,0 +1,275 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d_blocked.js:
```diff
@@ -0,0 +1,275 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d_blocked.js:
```diff
@@ -0,0 +1,275 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d_blocked.js:
```diff
@@ -0,0 +1,275 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/4d_blocked.js:
```diff
@@ -0,0 +1,275 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d.js:
```diff
@@ -0,0 +1,245 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d.js:
```diff
@@ -0,0 +1,245 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d.js:
```diff
@@ -0,0 +1,245 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d.js:
```diff
@@ -0,0 +1,245 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d.js:
```diff
@@ -0,0 +1,245 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d_blocked.js:
```diff
@@ -0,0 +1,298 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d_blocked.js:
```diff
@@ -0,0 +1,298 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d_blocked.js:
```diff
@@ -0,0 +1,298 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d_blocked.js:
```diff
@@ -0,0 +1,298 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/5d_blocked.js:
```diff
@@ -0,0 +1,298 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d.js:
```diff
@@ -0,0 +1,263 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d.js:
```diff
@@ -0,0 +1,263 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d.js:
```diff
@@ -0,0 +1,263 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d.js:
```diff
@@ -0,0 +1,263 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d.js:
```diff
@@ -0,0 +1,263 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d_blocked.js:
```diff
@@ -0,0 +1,321 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d_blocked.js:
```diff
@@ -0,0 +1,321 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d_blocked.js:
```diff
@@ -0,0 +1,321 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d_blocked.js:
```diff
@@ -0,0 +1,321 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/6d_blocked.js:
```diff
@@ -0,0 +1,321 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d.js:
```diff
@@ -0,0 +1,279 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d.js:
```diff
@@ -0,0 +1,279 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d.js:
```diff
@@ -0,0 +1,279 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d.js:
```diff
@@ -0,0 +1,279 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d.js:
```diff
@@ -0,0 +1,279 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d_blocked.js:
```diff
@@ -0,0 +1,344 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d_blocked.js:
```diff
@@ -0,0 +1,344 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d_blocked.js:
```diff
@@ -0,0 +1,344 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d_blocked.js:
```diff
@@ -0,0 +1,344 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/7d_blocked.js:
```diff
@@ -0,0 +1,344 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d.js:
```diff
@@ -0,0 +1,295 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d.js:
```diff
@@ -0,0 +1,295 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d.js:
```diff
@@ -0,0 +1,295 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d.js:
```diff
@@ -0,0 +1,295 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d.js:
```diff
@@ -0,0 +1,295 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d_blocked.js:
```diff
@@ -0,0 +1,367 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d_blocked.js:
```diff
@@ -0,0 +1,367 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d_blocked.js:
```diff
@@ -0,0 +1,367 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d_blocked.js:
```diff
@@ -0,0 +1,367 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/8d_blocked.js:
```diff
@@ -0,0 +1,367 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d.js:
```diff
@@ -0,0 +1,311 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d.js:
```diff
@@ -0,0 +1,311 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d.js:
```diff
@@ -0,0 +1,311 @@
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {boolean} isRowMajor - boolean indicating if provided arrays are in row-major order
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d.js:
```diff
@@ -0,0 +1,311 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d.js:
```diff
@@ -0,0 +1,311 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d_blocked.js:
```diff
@@ -0,0 +1,391 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray via loop blocking.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray via loop blocking.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d_blocked.js:
```diff
@@ -0,0 +1,391 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d_blocked.js:
```diff
@@ -0,0 +1,391 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d_blocked.js:
```diff
@@ -0,0 +1,391 @@
+* var ox = 0;
+* var oy = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/9d_blocked.js:
```diff
@@ -0,0 +1,391 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/factory.js:
```diff
@@ -0,0 +1,172 @@
+// MAIN //
+
+/**
+* Return a function for applying a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Return a function for applying a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/factory.js:
```diff
@@ -0,0 +1,172 @@
+*
+* @param {Function} [fcn] - wrapper for a one-dimensional strided array function
+* @param {Options} [options] - function options
+* @param {boolean} [options.strictTraversalOrder=false] - boolean specifying whether to require that element traversal match the memory layout of an input ndarray
```
```suggestion
* @param {boolean} [options.strictTraversalOrder=false] - boolean specifying whether to require that element traversal match the memory layout of an output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/factory.js:
```diff
@@ -0,0 +1,172 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/factory.js:
```diff
@@ -0,0 +1,172 @@
+	return f;
+
+	/**
+	* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
	* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/factory.js:
```diff
@@ -0,0 +1,172 @@
+	}
+
+	/**
+	* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
	* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/index.js:
```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+/**
+* Apply a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Apply a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/index.js:
```diff
@@ -0,0 +1,153 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/index.js:
```diff
@@ -0,0 +1,153 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main.js:
```diff
@@ -0,0 +1,157 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main.js:
```diff
@@ -0,0 +1,157 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main.js:
```diff
@@ -0,0 +1,157 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main_factory.js:
```diff
@@ -0,0 +1,371 @@
+// MAIN //
+
+/**
+* Returns a function for applying a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Returns a function for applying a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main_factory.js:
```diff
@@ -0,0 +1,371 @@
+*
+* @private
+* @param {Options} [options] - function options
+* @param {boolean} [options.strictTraversalOrder=false] - boolean specifying whether to require that element traversal match the memory layout of an input ndarray
```
```suggestion
* @param {boolean} [options.strictTraversalOrder=false] - boolean specifying whether to require that element traversal match the memory layout of an output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main_factory.js:
```diff
@@ -0,0 +1,371 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main_factory.js:
```diff
@@ -0,0 +1,371 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main_factory.js:
```diff
@@ -0,0 +1,371 @@
+	return nullaryStrided1d;
+
+	/**
+	* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
	* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main_factory.js:
```diff
@@ -0,0 +1,371 @@
+	* @param {IntegerArray} dims - list of dimensions to which to apply a strided array function
+	* @param {Options} [options] - function options
+	* @throws {Error} arrays must have the expected number of dimensions
+	* @throws {RangeError} dimension indices must not exceed input ndarray bounds
```
```suggestion
	* @throws {RangeError} dimension indices must not exceed output ndarray bounds
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/main_factory.js:
```diff
@@ -0,0 +1,371 @@
+	* @param {Options} [options] - function options
+	* @throws {Error} arrays must have the expected number of dimensions
+	* @throws {RangeError} dimension indices must not exceed input ndarray bounds
+	* @throws {RangeError} number of dimension indices must not exceed the number of input ndarray dimensions
```
```suggestion
	* @throws {RangeError} number of dimension indices must not exceed the number of output ndarray dimensions
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/nd.js:
```diff
@@ -0,0 +1,173 @@
+// MAIN //
+
+/**
+* Applies a one-dimensional strided array function to a list of specified dimensions in an input ndarray.
```
```suggestion
* Applies a one-dimensional strided array function to a list of specified dimensions in an output ndarray.
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/nd.js:
```diff
@@ -0,0 +1,173 @@
+* @param {Array<Object>} arrays - ndarrays
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
```
```suggestion
* @param {IntegerArray} stridesX - loop dimension strides for the output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/nd.js:
```diff
@@ -0,0 +1,173 @@
+* @param {Array<Object>} views - initialized ndarray-like objects representing sub-array views
+* @param {NonNegativeIntegerArray} shape - loop dimensions
+* @param {IntegerArray} stridesX - loop dimension strides for the input ndarray
+* @param {Object} strategyX - strategy for marshaling data to and from an input ndarray view
```
```suggestion
* @param {Object} strategyX - strategy for marshaling data to and from an output ndarray view
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/nd.js:
```diff
@@ -0,0 +1,173 @@
+* // Define the index offsets:
+* var ox = 0;
+*
+* // Create an input ndarray-like object:
```
```suggestion
* // Create an output ndarray-like object:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/nd.js:
```diff
@@ -0,0 +1,173 @@
+*     }
+* ];
+*
+* // Define an input strategy:
```
```suggestion
* // Define an output strategy:
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/strategy.js:
```diff
@@ -0,0 +1,262 @@
+* Returns an input ndarray.
+*
+* @private
+* @param {ndarrayLike} x - input ndarray
+* @returns {ndarrayLike} input ndarray
```
```suggestion
* @param {ndarrayLike} x - output ndarray
* @returns {ndarrayLike} output ndarray
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/strategy.js:
```diff
@@ -0,0 +1,262 @@
+* Returns a function for reshaping input ndarrays which have the same data type, shape, and strides as a provided ndarray.
+*
+* @private
+* @param {ndarrayLike} x - input ndarray
+* @param {string} x.dtype - input ndarray data type
+* @param {Collection} x.data - input ndarray data buffer
+* @param {NonNegativeIntegerArray} x.shape - input ndarray shape
+* @param {IntegerArray} x.strides - input ndarray strides
+* @param {NonNegativeInteger} x.offset - input ndarray index offset
+* @param {string} x.order - input ndarray memory layout
```
```suggestion
* @param {ndarrayLike} x - output ndarray
* @param {string} x.dtype - output ndarray data type
* @param {Collection} x.data - output ndarray data buffer
* @param {NonNegativeIntegerArray} x.shape - output ndarray shape
* @param {IntegerArray} x.strides - output ndarray strides
* @param {NonNegativeInteger} x.offset - output ndarray index offset
* @param {string} x.order - output ndarray memory layout
```

headlessNode commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/package.json:
```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@stdlib/ndarray/base/nullary-strided1d",
+  "version": "0.0.0",
+  "description": "Apply a one-dimensional strided array function to a list of specified dimensions in an input ndarray.",
```
```suggestion
  "description": "Apply a one-dimensional strided array function to a list of specified dimensions in an output ndarray.",
```

headlessNode committed:
Merge branch 'stdlib-js:develop' into nullary-strided1d

headlessNode commented:
@kgryte let me know if should go ahead and apply the suggestions above. The suggested change for all is to change `input` in the description to `output`, since the function is `nullary`.

kgryte commented:
@headlessNode How about just saying `in an ndarray` and omitting `input/output` entirely?

kgryte commented:
The input/output qualifier is less applicable here, as the input array is really treated as both.

headlessNode commented:
@kgryte Makes sense.

headlessNode committed:
refactor: docs description

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: passed
  - task: lint_repl_help
    status: passed
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte commented on lib/node_modules/@stdlib/ndarray/base/nullary-strided1d/lib/2d.js:
```diff
@@ -0,0 +1,197 @@
+	for ( i1 = 0; i1 < S1; i1++ ) {
+		for ( i0 = 0; i0 < S0; i0++ ) {
+			setViewOffsets( views, iv );
+			v[ 0 ] = strategyX( views[ 0 ] );
```
The main issue with the current implementation is you need to more closely follow `unary-strided1d` in terms of having **both** an input **and** output strategy. It is not enough to simply have an input strategy. 

Why? Because the input strategy could be to copy data to a workspace. If the strided array function mutates that data (e.g., sorting), then we need to copy that data back from the workspace to the original input array. Right now, your implementation does not handle that. You only handle the input strategy, but not the output.

In short, "nullary" effectively assumes that a provided ndarray is both an input and an output array.

Accordingly, I suggest copying, more or less, the `strategy.js` file from `unary-strided1d` and then updating all your loop kernels accordingly.

kgryte commented:
@headlessNode Also, as an aside, I see that you went through each file and added various change requests (over 100 of them). That follow-up is certainly appreciated; however, I might suggest for repeated changes like that, just doing that review in VSCode and pushing up the changes all in one go, potentially by using find-and-replace. In this case, GitHub was not particularly adept at determining that you had resolved all those change requests, requiring me (or someone else) to mark them as "resolved".

headlessNode committed:
refactor: apply suggestions from code review

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

headlessNode committed:
fix: apply suggestions from code review

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte committed:
docs: update comments

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte committed:
docs: fix examples

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte committed:
fix: add missing imports

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte committed:
docs: update comment

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte committed:
docs: update comments

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte committed:
docs: update copy

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: passed
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte reviewed (dismissed):
LGTM after some clean-up.

kgryte committed:
docs: fix examples

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte committed:
docs: fix example

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---
